### PR TITLE
Pod Annotation.

### PR DESCRIPTION
**Pull request by sfxworks**
While using linkerd:
linkerd's tap service listens on 4190
```
time="2020-07-21T19:57:29Z" level=info msg="running version stable-2.8.1"
[     0.10753299s]  INFO linkerd2_proxy: Admin interface on 0.0.0.0:4191
[     0.10769128s]  INFO linkerd2_proxy: Inbound interface on 0.0.0.0:4143
[     0.10771513s]  INFO linkerd2_proxy: Outbound interface on 127.0.0.1:4140
[     0.10773837s]  INFO linkerd2_proxy: Tap interface on 0.0.0.0:4190
```
This conflicts with dovecot, preventing the pod from starting.
```
kubectl logs -f -n mail mail-mailu-dovecot-5dfb597c97-7gtsh -c admin
Error: service(managesieve-login): listen(*, 4190) failed: Address in use
Jul 21 19:57:49 master: Error: service(managesieve-login): listen(*, 4190) failed: Address in use
Fatal: Failed to start listeners
Jul 21 19:57:49 master: Fatal: Failed to start listeners
```
The solution would be to add an annotation to the deployment's pod.
https://linkerd.io/2/reference/proxy-configuration/
However, there was no ability to do so with the current chart's configuration.

PR Allows for adding annotations to all deployment pods. 

Also added an example podAffinity value as a storage class of RWO will need this to not have pods that share the same PVC stuck in pending.

### CODE REVIEW

**Comment by micw** [comment]
Hi,

would you call the variable "podAnnotations" rather than annotations and sync with the https://github.com/Mailu/helm-charts/pull/68 merge?

Kind regards,
Michael.

File: mailu/values.yaml [hunk 1]
```diff
@@ -31,6 +31,15 @@ tolerations: {}
 # Affinity for pod assignment
 # Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 affinity: {}
+#  podAffinity:
```
Comment (micw) [inline mailu/values.yaml:34]: Please remove this unrelated change from the PR. Thank you.

**Comment by fastlorenzo** [comment]
Fixed in #193
